client/additional-methods-setup/upe-preview-methods-selector/setup-complete-task.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useEffect, useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import CollapsibleBody from '../wizard/collapsible-body';
+import WizardTaskItem from '../wizard/task-item';
+import WizardTaskContext from '../wizard/task/context';
+
+const SetupComplete = () => {
+	const { isActive } = useContext( WizardTaskContext );
+
+	const { updateOptions } = useDispatch( 'wc/admin/options' );
+
+	useEffect( () => {
+		if ( ! isActive ) {
+			return;
+		}
+
+		updateOptions( {
+			// eslint-disable-next-line camelcase
+			wcpay_additional_methods_setup_completed: 'yes',
+		} );
+
+		// Set the local `isSetupCompleted` to `yes` so that task appears completed on the list.
+		// Please note that marking an item as "completed" is different from "dismissing" it.
+		window.wcpayAdditionalMethodsSetup.isSetupCompleted = 'yes';
+	}, [ isActive, updateOptions ] );
+
+	return (
+		<WizardTaskItem
+			title={ __( 'Enjoy the new features', 'woocommerce-payments' ) }
+			index={ 3 }
```

Comment:
There currently are 2 tasks, but I named this "3" on purpose, because in the end we'll have 3 tasks. Adjustments to be made in https://github.com/Automattic/woocommerce-payments/issues/2170